aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -1,19 +1,17 @@
 ---
-remote_endpoint: https://fullnode.mainnet.aptoslabs.com
-name: "TBD"
+remote_endpoint: ~
+name: "v1.21.0"
 proposals:
   - name: proposal_1_upgrade_framework
     metadata:
-      title: "Multi-step proposal to upgrade mainnet framework, version TBD"
-      description: "This includes changes in (TBA: URL to changes)"
+      title: "Multi-step proposal to upgrade mainnet framework, version v1.21.0"
+      description: "This includes changes in https://github.com/aptos-labs/aptos-core/releases/tag/aptos-node-v1.21.1-rc and https://github.com/aptos-labs/aptos-core/releases/tag/aptos-node-v1.21.0-rc"
     execution_mode: MultiStep
     update_sequence:
       - Gas:
-          new: current
+          old: https://raw.githubusercontent.com/aptos-labs/aptos-networks/main/gas/v1.20.0.json
+          new: https://raw.githubusercontent.com/aptos-labs/aptos-networks/main/gas/v1.21.1.json
```

Comment:
I think you need to change this to 1.21.0 too :)

Reply:
This is generated from the latest v1.21.1 binary. Does the name of gas schedule version named after the binary version or node version doesn't matter? cc: @sherry-x